pan3d/utils/globe.py

```diff
@@ -64,6 +65,32 @@ def get_globe_texture():
     texture.InterpolateOn()
 
     return texture
+"""
+
+
+def get_globe_textures():
```

Comment:
should you cache those textures that never change instead of creating them every time you call that method?

Reply:
They are cached, the call to this method only happens once in the `Globe Explorer` so instead of caching it in the utils class they are used as a class member in the explorer. This method won't be called every time the user changes the texture. Does that still not feel okay?